website/lib/labels/classic/ClassicTemplate.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class ClassicTemplate extends PngTemplate {
+    public function getName() {
+        return 'Classic :: filips';
```

Comment:
this one render amost black on black on my side

![image](https://user-images.githubusercontent.com/3100576/59266119-3027eb80-8c47-11e9-9f93-481e7e3d2d29.png)

(but this was already the case on prod)

Reply:
I noticed, but I don't really know what to do with it. If you click "Print" in browser, that it's completely fine and good. So I consider the label itself as good enough.
If I disable some transparency flag during image generation, it starts to look ugly. Maybe I can play with source PNG, I can try... but shouldn't be a stopper anyway...

Reply:
Fixed, added white background overlay

Reply:
excellent! 
we feels less "buggy-like" behavior